eventmesh-dashboard-core/pom.xml

```diff
@@ -52,6 +52,24 @@
             <version>0.0.1-SNAPSHOT</version>
         </dependency>
 
+        <!-- eventmesh -->
+        <dependency>
+            <groupId>org.apache.eventmesh</groupId>
+            <artifactId>eventmesh-sdk-java</artifactId>
+            <version>1.10.0-release</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.eventmesh</groupId>
+            <artifactId>eventmesh-common</artifactId>
+            <version>1.10.0-release</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.logging.log4j</groupId>
+                    <artifactId>log4j-slf4j-impl</artifactId>
+                </exclusion>
```

Comment:
`eventmesh-sdk-java` artifact will propagate `eventmesh-common`, so introducing `eventmesh-sdk-java` is enough.